### DISCUSSION TITLE
Info on usage - install

### DISCUSSION THREAD

**Discussion by docop** [devfinwiz/Fin-Maestro#29] [Q&A]
Hi
 So this project look quite interesting, but i would like to know.. is it made to be run in docker or is there a running website live with it running ? As i haven't find any detail on how to run this. Like i will need to point to a local ip:portxyz  .. ?
Thanks for tips

**Comment by devfinwiz** [comment]
> Hi
>  So this project look quite interesting, but i would like to know.. is it made to be run in docker or is there a running website live with it running ? As i haven't find any detail on how to run this. Like i will need to point to a local ip:portxyz  .. ?
> Thanks for tips

Hey, 
The project isn't ready to be deployed yet as the work is in progress. We have our UI in Vue.Js and you can find it's code in branch withFE. 
But as it is not ready to be deployed yet due to pending tasks, we are also creating an alternative UI using Gradio that can help execute our project within minutes. You can execute the driver.py script in CommonFEGradio directory to get hold of 3 modules. Support for others will be there soon. 


Also, containerization of Fin-Maestro is one of the targets for the future. I presently have too much on my plate so I am unable to dedicate a significant amount of time to this project. 

I'll make the necessary updates in the readme regarding the same in a few days. Also do star the project in case you find it interesting and feel free to contribute to it if you can. ✨
